firmware/screen/src/ui/screen_pocket_technician.cpp

```
@@ -0,0 +1,495 @@
+
+// ---- Per-screen ctx ------------------------------------------------------
+
+struct pt_ctx_t {
+    lv_obj_t* meta_track;
+    lv_obj_t* meta_car;
+    lv_obj_t* list_col;        // scrollable column
+    lv_obj_t* placeholder_lbl; // "Loading…" / "No setups found"
+    lv_obj_t* active_row_obj;  // row being loaded (for gold pulse)
+    lv_timer_t* pulse_timer;
+    int       pulse_steps_left;
+    char      pending_name[48];   // setup the user just tapped
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)